src/main.c

```diff
@@ -2652,7 +2659,7 @@ int main(int argc, char *argv[]) {
     }
 
   } else {
-    int num;
+    int num = 0;
```

Comment:
is this needed?

Reply:
I think it is, right?

Reply:
MSVC warning that the variable isn't initialized